pkg/controllers/resources/volumesnapshots/volumesnapshotcontents/syncer.go

```diff
@@ -177,16 +164,30 @@ func (s *volumeSnapshotContentSyncer) Sync(ctx *synccontext.SyncContext, pObj cl
 			}
 			return ctrl.Result{}, err
 		}
+	}
 
-		updatedPv := s.translateUpdate(ctx, vVSC, pVSC)
-		if updatedPv != nil {
-			ctx.Log.Infof("update physical VolumeSnapshotContent %s, because spec or annotations have changed", updatedPv.Name)
-			translator.PrintChanges(pVSC, updatedPv, ctx.Log)
-			err := ctx.PhysicalClient.Update(ctx, updatedPv)
-			if err != nil {
-				return ctrl.Result{}, err
-			}
+	// patch objects
+	patch, err := patcher.NewSyncerPatcher(ctx, pVSC, vVSC)
+	if err != nil {
+		return ctrl.Result{}, fmt.Errorf("new syncer patcher: %w", err)
+	}
+	defer func() {
+		if err := patch.Patch(ctx, pVSC, vVSC); err != nil {
+			retErr = utilerrors.NewAggregate([]error{retErr, err})
 		}
+	}()
+
+	// update virtual object
+	s.translateUpdateBackwards(pVSC, vVSC)
+
+	// update virtual status
+	vVSC.Status = pVSC.Status.DeepCopy()
+
+	// update host object
+	if vVSC.Annotations == nil || vVSC.Annotations[constants.HostClusterVSCAnnotation] == "" {
```

Comment:
```suggestion
	if vVSC.Annotations[constants.HostClusterVSCAnnotation] == "" {
```
would be equivalent I think